docs/Build/grestsvcs.json

```diff
@@ -50,6 +50,23 @@
         }
       }
     },
+    "/genesis": {
+      "post": {
+        "tags": [
+          "Blockchain Queries"
+        ],
+        "summary": "Genesis parameters used to create the chain",
```

Comment:
Not sure if we want to have it standard as the other queries: "Get the genesis parameters..."?

Reply:
done